packages/grafana-ui/src/components/TimePicker/TimeRangePicker.tsx

```diff
@@ -224,3 +180,44 @@ const formattedRange = (value: TimeRange, timeZone?: TimeZone) => {
 };
 
 export const TimeRangePicker = withTheme(UnthemedTimeRangePicker);
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  return {
+    container: css`
+      position: relative;
+      display: flex;
+      flex-flow: column nowrap;
+      vertical-align: middle;
+    `,
+    buttons: css`
+      display: flex;
+    `,
+    caretIcon: css`
+      margin-left: ${theme.spacing.xs};
+    `,
+    clockIcon: css`
+      margin-left: ${theme.spacing.xs};
+      margin-right: ${theme.spacing.xs};
+    `,
+    noRightBorderStyle: css`
+      label: noRightBorderStyle;
```

Comment:
```suggestion
```

Reply:
Torkel, here I am curious about your thoughts. When using stylesFactory, some of us sometimes use label for styles and sometimes not. And for the label, we sometimes use camelCase and sometimes kebab-case and there doesn’t seem to be any particular pattern. Do you have any preference?

Reply:
could remove all these styles, no longer used :)